liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/ServerXmlDocument.java

```diff
@@ -88,15 +137,15 @@ public static boolean isFoundWebApplication(String filePath, File configDirector
                         if (isValidURL(locValue)) {
```

Comment:
From line 136 which is not from this pull request, you should check or `http:` and `https:` and not just `http`

```
                    if (locValue.startsWith("http")) {
                        if (isValidURL(locValue)) {
                            URL url = new URL(locValue);
                            URLConnection connection = url.openConnection();
                            incDoc = docBuilder.parse(connection.getInputStream());
                        }
                    }
```